fe/fe-core/src/main/java/org/apache/doris/service/ExecuteEnv.java

```diff
@@ -28,10 +33,55 @@ public class ExecuteEnv {
     private ConnectScheduler scheduler;
     private long startupTime;
 
+    public static class DiskInfo {
+        private String dirType;
```

Comment:
This class is used elsewhere and should be put in a class file, with `toThrift` and `fromThrift` as member method